src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -128,6 +128,9 @@ public function handleError($type, $msg, $file, $line, $context = [])
         if ($deprecation->isMuted()) {
             return null;
         }
+        if ($this->getConfiguration()->isBaselineDeprecation($deprecation)) {
```

Comment:
I think this check needs to be skipped when we are regenerating the baseline. Otherwise, any deprecation which was in the previous baseline file would be removed from the file even though it is still needed.

Reply:
Nice catch. I've fixed this in how we initialise the configuration and added extra tests for file existence.